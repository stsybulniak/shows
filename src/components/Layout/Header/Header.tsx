import { FC } from 'react';
import './Header.scss';

const Header: FC = () => <header className='Header'>Header</header>;

export default Header;
