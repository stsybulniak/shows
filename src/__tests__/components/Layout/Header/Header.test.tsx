import { render } from '@testing-library/react';
import Header from '../../../../components/Layout/Header/Header';

describe('<Header />', () => {
  it('Header matches snapshot', () => {
    const { baseElement } = render(<Header />);
    expect(baseElement).toMatchSnapshot();
  });
})
