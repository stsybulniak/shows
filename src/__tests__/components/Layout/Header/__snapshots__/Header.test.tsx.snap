// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Header matches snapshot 1`] = `
<body>
  <div>
    <header
      class="Header"
    >
      Header
    </header>
  </div>
</body>
`;
